en/tutorials-and-examples/cms/installation.rst

```diff
@@ -51,7 +51,7 @@ in the **cms** directory of the current working directory:
 If you downloaded and ran the `Composer Windows Installer
 <https://getcomposer.org/Composer-Setup.exe>`_, then type the following line in
 your terminal from your installation directory (ie.
-C:\\wamp\\www\\dev\\cakephp3):
+C:\\wamp\\www\\dev\\cakephp):
```

Comment:
wouldnt we call this `project` or better `app` usually?

Reply:
I also wonder why this shows specific windows topics, I would assume a more generic approach (using just `/my/path/to/app` etc)

Reply:
anything but cakephp3 is good for me :-)